modules/api-robot-deleteOrgRobotFederation.adoc

```diff
@@ -0,0 +1,31 @@
+= deleteOrgRobotFederation 
+
+Delete a federation configuration for the specified organization robot.
+
+[discrete]
+== DELETE /api/v1/organization/{orgname}/robots/{robot_shortname}/federation
```

Comment:
Never see Quay new UI use the delete method, send to @syed to double confirm.

Reply:
Will hide for now!